docs/react/list-other.mdx

```diff
@@ -4,8 +4,8 @@ sidebar_position: 5
 
 # Display media of other peers
 
-To access data and media of other peers, use the `usePeers` hook.
-It returns two properties, `remotePeers` and `localPeer`.
+To access data and media of other peers, use the [`usePeers`](/api/web/functions/usePeers) hook.
+It returns two properties, [`remotePeers`](/api/web/functions/usePeers) and [`localPeer`](/api/web/functions/usePeers).
```

Comment:
**suggestion:** It would be nice if the links to `remotePeers` and `localPeer` actually pointed to the type definition, since the page for `usePeers` only says that it returns `UsePeersResult` without saying anything about what is `UsePeersResult`

Reply:
now it does